docs/tutorials/aws/AIConnectorHelper.ipynb

```diff
@@ -0,0 +1,1069 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5f2363c3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "from botocore.exceptions import BotoCoreError\n",
+    "import json\n",
+    "import requests\n",
+    "from requests.auth import HTTPBasicAuth\n",
+    "from requests_aws4auth import AWS4Auth\n",
+    "import time\n",
+    "\n",
```

Comment:
these "\n" are not necessary too?

Reply:
It's necessary to make the code more readable by adding an empty line